Dockerfile.efdtransform

```diff
@@ -0,0 +1,46 @@
+# Define the LSST version
+ARG OBS_LSST_VERSION=w_2024_47
+FROM lsstsqre/centos:7-stack-lsst_distrib-${OBS_LSST_VERSION}
+
+# Set user
+USER lsst
+
+# Install dependencies
+RUN source loadLSST.bash && mamba install -y aiokafka httpx && \
+    pip install \
+    kafkit==0.2.1 \
+    lsst_efd_client==0.12.0
+
+# Copy the Python module
+COPY --chown=lsst:lsst python /opt/lsst/software/stack/python
+
+# Create and populate the data directory
+RUN mkdir -p /opt/lsst/software/stack/data
+COPY --chown=lsst:lsst tmp/efd_transform/*.db /opt/lsst/software/stack/data/
+
+# Environment variables
+ENV CONFIG_FILE="/opt/lsst/software/stack/python/lsst/consdb/efd_transform/config_LATISS.yml" \
```

Comment:
This and `INSTRUMENT` will have to be overridden for LSSTCam deployments, I presume.

I think this is easier to maintain as a sequence of ENV lines rather than using backslashes.

Reply:
Yes, all env variables can be overridden by Kubernetes manifests or direct function call.
I will change the backslashes by ENV lines.